sdk/core/core-util/src/checkEnvironment.ts

```diff
@@ -74,6 +74,11 @@ export const isNode =
   !isDeno &&
   !isBun;
 
+/**
+ * A constant that indicates whether the environment the code is running is a Node.JS compatible environment.
+ */
+export const isNodeCompatible = isNode || isDeno || isBun;
```

Comment:
`isNodeLike` might be more idiomatic - thoughts?

Reply:
Changed to `isNodeLike`